app/src/main/java/vn/framgia/phamthehung/soundcloud/ui/more/MoreFragment.java

```diff
@@ -0,0 +1,76 @@
+package vn.framgia.phamthehung.soundcloud.ui.more;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.BottomSheetDialogFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import vn.framgia.phamthehung.soundcloud.R;
+import vn.framgia.phamthehung.soundcloud.data.model.Track;
+
+public class MoreFragment extends BottomSheetDialogFragment {
+    public static final String TRACK_KEY = "TRACK_KEY";
+    public static final int NUMBER_1 = 1000;
+    public static final int NUMBER_2 = 1000000;
+    private Track mTrack;
+    private TextView mTextTrack;
+    private TextView mTextArtist;
+    private ImageView mImageTrack;
+    private TextView mTextCountFavorite;
+    private TextView mPlayMore;
+    private TextView mTextComment;
+    private TextView mTextDownload;
+    private ImageView mImageDownload;
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_more, container, false);
+        initView(view);
+        getData(view);
+        return view;
+    }
+
+    private void getData(View view) {
+        Bundle bundle = getArguments();
+        mTrack = bundle.getParcelable(TRACK_KEY);
+        mTextTrack.setText(mTrack.getTitle());
+        mTextArtist.setText(mTrack.getArtist());
+        Glide.with(view).load(mTrack.getArtworkUrl()).into(mImageTrack);
+        mTextCountFavorite.setText(mTrack.getLikesCount()/NUMBER_1 + getString(R.string.msg_k));
```

Comment:
48 -> 51 không Cộng String

Reply:
Vì số lượt yêu thích và lượt nghe nhiều nên em đã chia cho 1000 và 1000000 thay vào đó là K và Tr thì không cộng String thì phải làm thế nào ạ

Reply:
Em tìm cách gán các String vào không phải bằng toán tử + là được.
Có thể dùng StringFormat hoặc StringBuilder